### DISCUSSION TITLE
ninja: build stopped: subcommand failed

### DISCUSSION THREAD

**Discussion by zhang-qiang-github** [su2code/SU2#1871] [General]
I am building `su2` in my compute (`win10`), and I failed:

![image](https://user-images.githubusercontent.com/29271051/210540722-d4430ae6-8a5b-4e33-bea0-5ccf79aa4c7e.png)

Then, I restart `.\ninja.exe -C build install`, and the compiling is continued.

Until `[0/318] Compiling C++ object externals/tecio/teciosrc/libtechio.a.p/NodeToElemMap_s.cpp.obj`, I can not continue the compiling.

How can I finish this compiling?

**Comment by pcarruscag** [comment]
Seems to be a problem in an external library, you either need CXXFLAGS="-fpermissive" (env variable before running meson.py), or to disable that library with -Denable-tecio=false

**Reply by zhang-qiang-github** [reply]
Thanks for kindly reply. I hava another question. I want to simulate the blood flow in aorta or artery with SU2. I want to know the blood velocity, pressure on wall, wall shear stress. Is there any similar example?
